Sprint Planning Docs/Sprint Planning#2.md

```diff
@@ -0,0 +1,54 @@
+# Sprint Planning #2
+
+**Trello Board**: https://trello.com/b/Do4WCnFy/teachers-pet-🐢
+
+# Project Summary
+
+Teacher's Pet is an app aimed at streamlining the office hours process. The goal of this app is to help instructors best help their students during office hours with ease in an effective, efficient manner. 
+
+# Tasks Completed
+
+**Ashley Valdez**
+- Got 1 TA To Test Our App 
+    - Bhavye Jain, CS TA @ UCSB.
+- Thinking of talking to Butner about having learning assistants test Teacher’s Pet.
+- [Contributed Frameworks] (https://www.notion.so/ashleys-workspace/Milestone-1-f9ac176336c24f648876866ccb21ecd0) that Teacher's Pet may require.
```

Comment:
In the future, provide links that take you directly to the source of information. As an example, here you can link the [Frameworks table](https://www.notion.so/ashleys-workspace/Milestone-1-f9ac176336c24f648876866ccb21ecd0?pvs=4#6a381e924a894da0800643bec5e4ab05) of the Milestone Notion block. You can get the link in Notion by hovering over the left side of a title, table, picture, etc., then clicking the 6 dots on the side, and clicking **Copy link to block**.